packages/ifc-import-service/src/ifc_importer/client.py

```diff
@@ -0,0 +1,26 @@
+from specklepy.core.api.client import (  # pyright: ignore[reportMissingTypeStubs]
+    SpeckleClient,
+)
+
+from src.ifc_importer.domain import FileimportPayload
```

Comment:
This import is giving me some trouble running the ifc-import-service docker image locally.

```
Traceback (most recent call last):

  File "/app/main.py", line 10, in <module>

    from ifc_importer.job_manager import job_manager

  File "/app/lib/python3.13/site-packages/ifc_importer/job_manager.py", line 16, in <module>

    from ifc_importer.client import setup_client

  File "/app/lib/python3.13/site-packages/ifc_importer/client.py", line 5, in <module>

    from src.ifc_importer.domain import FileimportPayload

ModuleNotFoundError: No module named 'src'
```